src/common/hash64.go

```diff
@@ -0,0 +1,12 @@
+package common
+
+import "hash/fnv"
+
+// Hash64 TODO
+func Hash64(data []byte) uint64 {
```

Comment:
package hash provides Hash64 type, https://golang.org/pkg/hash/#Hash64 , used in hash/fnv
It would be better to stick with that type instead of uint64 as result of Hash64 or define our own Hash64 type to make our code independent of Hash64 implementation we use.

Reply:
Not sure I understand. The Hash64 convenience function is just a wrapper around that Hash64 type and the Hash64 type you mention returns a uint64 for Sum():

```
type Hash64 interface {
        Hash
        Sum64() uint64
}
```